__tests__/app.test.js

```diff
@@ -108,4 +108,41 @@ describe('3. GET /api/articles', ()=>{
             expect(msg).toBe('Not Found');
         })
     });
+});
+
+describe('4. GET /api/articles/:article_id', ()=>{
+    test('responds with the requested article object', ()=>{
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body).toEqual({article: [{
```

Comment:
I would use toMatchObject instead here, it gives you flexibility over whether you ever change what this endpoint is asking for. toEqual must match exactly but toMatchObject doesn't have to include everything